Instagram Reels Bot/Helpers/IGComponentBuilder.cs

```diff
@@ -13,28 +13,28 @@ internal class IGComponentBuilder
         /// The Response from the Instagram processor.
         /// </summary>
         private InstagramProcessorResponse Response;
-        private string Requester;
+        private ulong RequesterId = 0;
         private bool RequesterIsKnown
         {
             get
             {
-                return !string.IsNullOrWhiteSpace(Requester);
+                return RequesterId != 0;
```

Comment:
Would `RequesterId!=null` work?

Reply:
`RequesterId` type is ulong, if `RequesterId!=null` it would warn `The result of the expression is always 'true' since a value of type 'ulong' is never equal to 'null' of type 'ulong?'`

Reply:
Ok, in that case, you should be able to leave it uninitialized and compare it to `default(uint)` (which I believe is also 0).